packages/apps/composer-app/src/main.tsx

```diff
@@ -87,6 +90,7 @@ const main = async () => {
       ThemeMeta,
       // Outside of error boundary so that updates are not blocked by errors.
       PwaMeta,
+      NativeMeta,
```

Comment:
you also need to add it to the `plugins` map, e.g. `[NativeMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-native')`

Reply:
Thanks!